### DISCUSSION TITLE
Questions about metrics we used to evaluate the generation of node embeddings

### DISCUSSION THREAD

**Discussion by HelloWorldLTY** [pyg-team/pytorch_geometric#6404] [General]
Hi, I wonder if there are any metrics we can use to evaluate the representation of embeddings for the total graph, for example, using the idea that nodes having edge connection tending to have similar expression therefore near distance. Thanks.

**Comment by rusty1s** [comment]
No expert, but what you can probably do is construct a k-NN graph of learned embeddings and measure the overlap between the k-NN graph and the original graph?
